themes/rad/layouts/index.html

```diff
@@ -98,19 +98,16 @@ <h2>Propose changes</h2>
     <div class="text">
       <h1>Radicle Stack</h1>
       <p>
-        Radicle utilities are built in a deterministic programming language
-        (the Radicle language). Each instance of a utility is a replicated
-        state machine (or simply <em>machine</em>) with its history distributed
-        and replicated via IPFS.
-        <br /><br />
-
-        The Radicle daemon materializes machines locally from their history,
-        allowing you to query it or submit new expressions.
-
-        <br /><br />
-
-        We have also built an IPFS-based git remote helper, so you can host
-        your git repository on IPFS as well.
+        Radicle utilities are written in the Radicle language, a deterministic Lisp dialect for creating P2P programs.
+        Each utility is a replicated state machine with state updates distributed and replicated via IPFS.
+      </p>
+      <p>
+        The Radicle daemon materializes machines from the local event log,
+        allowing you to query a machine's state or submit new expressions.
+      </p>
+      <p>
+        We've also built an git remote helper for IPFS, so you can also host
+        your git repository on the distributed web.
       </p>
```

Comment:
I feel that both of these have nothing to do with the illustration that is below. I thought the point here was to explain how the code collaboration stack works from a higher level.